app/Kernel.php

```diff
@@ -225,6 +228,50 @@ public function getCacheDir(): string
         return dirname(__DIR__) . '/var/cache/' . $this->environment . '/' . $this->request->getHost();
     }
 
+    private function setLocaleOnRequest(ContainerBuilder $container): void
+    {
+        $chunks = (array) explode('/', trim($this->request->getRequestUri(), '/'));
+        $defaultLocale = $container->getParameter('site.default_language');
+        $currentLocale = $chunks[0] ?? $defaultLocale;
+
+        $this->request->setLocale(strlen($defaultLocale) === 2 ? $currentLocale : $defaultLocale);
+        $this->request->setDefaultLocale($defaultLocale);
+    }
+
+    private function getSiteDomainForCurrentLocale(ContainerBuilder $container): string
+    {
+        if ($container->hasParameter('site.domain')) {
```

Comment:
What none of the parameters is defined?

Reply:
this one is there by default so it's always there unless you deliberetly change the config